src/en/space-station-14/roleplay-lore/proposals/Departmental-Operating-Procedure.md

```diff
@@ -0,0 +1,122 @@
+
+    idR[Under Red Alert]-->idCC3[Central Command];
+    style idR fill:#cd1c18,stroke:#333,stroke-width:4px
+    idCC3-->idC3[Captain];
+    idC3-->idHOS1[The Head of Security is in command over Security issues during red alert];
+    idHOS1-->idR1[All head of staff are equal below HOS];
+    end
+    subgraph Non-Standard Alerts
+
+    idY[Under Yellow Alert]-->idCC4[Central Command];
+    style idY fill:#ffd32c,stroke:#333,stroke-width:4px
+    idCC4-->idC4[Captain];
+    idC4-->idCE1[The Chief Engineer is in command over Engineering issues during yellow Alert];
+    idCE1-->idY1[All head of staff are equal below CE];
+
+    idV[Under Violet Alert]-->idCC5[Central Command];
+    style idV fill:#7f00ff,stroke:#333,stroke-width:4px
+    idCC5-->idC5[Captain];
+    idC5-->idCMO1[The Chief Medical Officer is in command over Medical issues during violet alert];
+    idCMO1-->idV1[All head of staff are equal below CMO];
+    end
+```
+
+### Security: 
+
+Security will follow the "Security Operating Procedure" (SOP), it will feature things like "Security Dresscode", "Optimal Proccessing Time Guideline", "Scene Response Guideline", "Evidence Handling and Collection Guideline", and "Interrogations 101" 
+
+I plan for these to add not only some interesting guidelines that will hopefully improve roleplay, but to also add lore. You may have noticed at this point but a lot of these are named like guidelines you would see in an office, and that is sort of my intention. I am hoping to add lore friendly Nanotrasen guidebooks for each department. With fun tidbits and interesting insights to the departments. 
+
+### Engineering: 
+
+Engineering will follow the "Engineering & Atmospherics Operating Procedure" (EAOP), it will feature things like "Station Modification Guidelines", "Departmental Repair Guidelines", and "Maintaining Clean Air"
+
+These guidelines will largely be a way to roleplay building things 'to code' as well as some more lore focused documentation like "Maintaining Clean Air" 
+
+### Cargo
+
+Cargo will follow the "Logistical Operating Procedure" (LOP), it will feature things like "Supply and Demand", "Cargo Handling Guidelines", and "Salvage Technician Safety Guide"
+
+These guidelines much like engineering will encourage roleplay within cargo by offering a set of guidelines to follow as well as adding some lore text with things like "Supply and Demand"
+
+### Medical
+
+Medical will follow the "Medical Operating Procedure" (MOP), it will feature things like "Medical Evaluation Guidelines", "Keeping Your Patients Happy", and "Psychology 101"
+
+### Research
+
+Research will follow the "Research Operating Procedure" (ROP), it will feature things like "Research and Develeopment: How not to blow up", "Research Operation Guidelines", and "Human Experimentation: An Art" 
+
+### Service
+
+Service will not have it's own Operating Procedure and will instead have guidelines and more lore focused text for each job I.E. Chef, Janitor, and Barkeep. 
```

Comment:
I mean, if you want it to be so, then so be it, but I find it rather strange that the most roleplay-focused department does not get their procedures that are, well, roleplay recommendations.

Reply:
That is a good point. I will work on something for service.